src/Sulu/Bundle/AdminBundle/Tests/Unit/Metadata/SchemaMetadata/SelectionPropertyMetadataMapperTest.php

```diff
@@ -0,0 +1,290 @@
+    {
+        $this->selectionPropertyMetadataMapper = new SelectionPropertyMetadataMapper();
+    }
+
+    public function testMapPropertyMetadata(): void
+    {
+        $propertyMetadata = new PropertyMetadata();
+        $propertyMetadata->setName('property-name');
+
+        $jsonSchema = $this->selectionPropertyMetadataMapper->mapPropertyMetadata($propertyMetadata)->toJsonSchema();
+
+        $this->assertEquals([
+            'name' => 'property-name',
+            'type' => 'array',
+            'items' => [
+                'required' => [],
+                'anyOf' => [
+                    ['type' => 'string', 'required' => []],
+                    ['type' => 'number', 'required' => []],
+                ],
+            ],
+            'uniqueItems' => true,
+        ], $jsonSchema);
+    }
+
+    public function testMapPropertyMetadataRequired(): void
+    {
+        $propertyMetadata = new PropertyMetadata();
+        $propertyMetadata->setName('property-name');
+        $propertyMetadata->setRequired(true);
+
+        $jsonSchema = $this->selectionPropertyMetadataMapper->mapPropertyMetadata($propertyMetadata)->toJsonSchema();
+
+        $this->assertEquals([
+            'name' => 'property-name',
+            'type' => 'array',
+            'items' => [
+                'required' => [],
+                'anyOf' => [
+                    ['type' => 'string', 'required' => []],
+                    ['type' => 'number', 'required' => []],
+                ],
+            ],
+            'minItems' => 1,
+            'uniqueItems' => true,
+        ], $jsonSchema);
+    }
+
+    public function testMapPropertyMetadataMinAndMax(): void
+    {
+        $propertyMetadata = new PropertyMetadata();
+        $propertyMetadata->setName('property-name');
+        $propertyMetadata->setParameters([
+            ['name' => 'min', 'value' => 3],
+            ['name' => 'max', 'value' => 5],
+        ]);
+
+        $jsonSchema = $this->selectionPropertyMetadataMapper->mapPropertyMetadata($propertyMetadata)->toJsonSchema();
+
+        $this->assertEquals([
+            'name' => 'property-name',
+            'type' => 'array',
+            'items' => [
+                'required' => [],
+                'anyOf' => [
+                    ['type' => 'string', 'required' => []],
+                    ['type' => 'number', 'required' => []],
+                ],
+            ],
+            'minItems' => 3,
+            'maxItems' => 5,
+            'uniqueItems' => true,
+        ], $jsonSchema);
+    }
+
+    public function testGetValidatedMinMaxValue(): void
+    {
+        $propertyMetadata = new PropertyMetadata();
+        $propertyMetadata->setName('property-name');
+        $propertyMetadata->setParameters([
+            ['name' => 'min', 'value' => 2],
+            ['name' => 'max', 'value' => 3],
+        ]);
+
+        $minMaxValue = SelectionPropertyMetadataMapper::getValidatedMinMaxValue($propertyMetadata);
```

Comment:
Why not using the public `mapPropertyMetadata` function here? This might cause the tests to break, if we refactor the `PropertyMetadataMapper`, although everything might still work fine.